webapi/src/main/resources/application.conf

```diff
@@ -324,8 +324,8 @@ app {
     }
 
     triplestore {
-        dbtype = "fuseki"
-        // dbtype = "graphdb"
+        // dbtype = "fuseki"
```

Comment:
Could you switch that back to fuseki?

Reply:
sorry!
- [ ] re-enable fuseki

Reply:
happens to me all the time ;-)

Reply:
done: 6ae2e757e2b4a7e7fc84198ed73fa030f044cc52